src/Apps/SaleAgreements/__tests__/SaleAgreementsApp.jest.tsx

```diff
@@ -0,0 +1,54 @@
+import { graphql } from "react-relay"
+import { SaleAgreementsApp_Test_Query } from "__generated__/SaleAgreementsApp_Test_Query.graphql"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { screen } from "@testing-library/react"
+import { SaleAgreementsAppFragmentContainer } from "Apps/SaleAgreements/SaleAgreementsApp"
+
+jest.unmock("react-relay")
+jest.mock("System/Router/useRouter", () => ({
+  useRouter: () => ({ match: { params: { id: "example" } } }),
+}))
+jest.mock("Components/MetaTags", () => ({
+  MetaTags: () => null,
+}))
+
+describe("SaleAgreementsApp", () => {
+  const { renderWithRelay } = setupTestWrapperTL<SaleAgreementsApp_Test_Query>({
+    Component: (props: any) => {
+      return <SaleAgreementsAppFragmentContainer viewer={props.viewer} />
+    },
+    query: graphql`
+      query SaleAgreementsApp_Test_Query @relay_test_operation {
+        viewer {
+          ...SaleAgreementsFilter_viewer
+        }
+      }
+    `,
+  })
+
+  it("renders correctly", () => {
+    renderWithRelay({
+      SaleAgreements: () => ({
+        saleAgreementsConnection: {
+          edges: [
+            {
+              node: {
+                internalID: "abc123",
+                displayStartAt: "2024-01-01",
+                displayEndAt: "2024-01-01",
+                published: true,
+                status: "CURRENT",
+                sale: {
+                  internalID: "xyz987",
+                  name: "Test Auction",
```

Comment:
@damassi - My understanding is this should stub out the relay response, but I'm still getting the auto-generated mock values. Tried to look at other examples and code, but I am unable to easily figure out what is wrong. Is anything sticking out?

Will add more tests once I get this resolved.

Reply:
When this happens I always end up slightly confused. I feel like this doesn't happen to the same extent in our other relay apps. 

My one suggestion would be to mock out the saleAgreementsConnection node type, whatever that might be, and see if that fixes it.